ACMAS/app/ACMAS/settings.py

```diff
@@ -123,22 +116,21 @@
 
 USE_TZ = True
```

Comment:
Should this be a flag?

Reply:
Should what be a flag?

Reply:
Should `USE_TZ` be an environment variable for consistency?

Reply:
Since USE_TZ isn't sensitive and doesn't need to change I'm not going to put it in an env file, but I did add a comment explaining what it does: "Stores datetime information in UTC"